react-native/react/menubar/index.render.desktop.js

```diff
@@ -79,34 +94,159 @@ class OpeningMessage extends Component {
   }
 }
 
+class FolderRow extends Component {
```

Comment:
This might become a generalized component used throughout the app (folders list, search results page, etc)